openam-ui/openam-ui-ria/src/main/resources/locales/ja/translation.json

```
@@ -0,0 +1,1292 @@
+{
+    "config" : {
+        "dates" : {
+            "monthNames" : "1月, 2月, 3月, 4月, 5月, 6月, 7月, 8月, 9月, 10月, 11月, 12月",
+            "monthNamesShort" : "1月, 2月, 3月, 4月, 5月, 6月, 7月, 8月, 9月, 10月, 11月, 12月",
+            "dayNames" : "日曜日, 月曜日, 火曜日, 水曜日, 木曜日, 金曜日, 土曜日",
+            "dayNamesShort" : "日, 月, 火, 水, 木, 金, 土"
+        },
+        "messages" : {
+            "AppMessages" : {
+                "invalidRealm": "レルムが存在しません。",
+                "duplicateRealm": "重複するレルムです。",
+                "deleteFail": "削除に失敗しました。",
+                "duplicateItem": "重複するアイテムです。",
+                "invalidItem": "不正なアイテムです。",
+                "changesSaved": "変更を保存しました。"
+            },
+            "CommonMessages" : {
+                "authenticationFailed" : "ユーザー名/パスワードの組み合わせが不正です。",
+                "serviceUnavailable" : "サービスが利用できません。",
+                "unknown" : "不明なエラー。システム管理者にお問い合わせください。",
+                "loggedIn" : "正常にログインしました。",
+                "errorFetchingData" : "ユーザーデータフェッチ中のエラー。",
+                "loggedOut" : "ログアウトしました。",
+                "loginTimeout" : "ログインセッションがタイムアウトになりました。再起動しています...",
```

`再起動しています...` -> `最初からやり直してください。`

---

I think that it is a reasonable proposal, thank you.
```suggestion
                "loginTimeout" : "ログインセッションがタイムアウトになりました。最初からやり直してください。",
```